lib/u3d/commands_generator.rb

```diff
@@ -189,6 +189,14 @@ def run
         end
       end
 
+      command :licenses do |c|
+        c.syntax = "u3d licenses>"
+        c.description = 'Prints Unity license information.'
+        c.action do |args, _options|
+          Commands.licenses(args: args)
```

Comment:
Idem, you could omit args if you don't need them further down

Reply:
I preferred to keep them to be similar to the other commands. The idea was to maybe add feature like add license later.

Reply:
Well not all of our commands have `args` as a parameter (list, available and dependencies don't for instance) and dependencies does not have `options` either. But it's really minor nitpicking, so do as you feel best